### ISSUE TITLE
Scality Deploy on Win2k12r2 VM

### ISSUE THREAD

**Issue by Mariosworld11** [scality/S3#117]
I am trying to deploy Scality on a Windows 2012R2 Virtual Machine but I am seeing the following errors when I install the DockerToolbox:

**Starting Kitematic fails**
Gets to 50% and gives following error: Error: connect ENOENT //./pipe/docker_engine 

**Starting Docker Quickstart Terminal gives the following error:**
Running pre-create checks...
Error with pre-create check: "This computer doesn't have VT-X/AMD-v enabled. Enabling it in the BIOS is mandatory"
Looks like something went wrong in step 'Checking if machine default exists'...
Press any key to continue... 

Here is what I am using:
Windows 2012R2 Datacenter Build 9600 (Virtual Machine)
                Hyper-V is not installed
vCenter 5.5.0
Installed Cyberduck-Installer-5.0.11.20753.exe 
Installed DockerToolbox-1.12.0.exe 
                Oracle Virtual Box was also installed

I have tried changing the CPU/MMU Virtualization settings on my VM from Automatic to:
-Use Intel VT-x/AMD-V for instruction set virtualization and software for MMU virtualization
AND also tried
-Use Intel VT-x/AMD-V for instruction set virtualization and Intel EPT/AMD RVI for MMU virtualization.

After making these changes I also uninstalled and re-installed the DockerToolbox. Please let me know if there is something else that I can try or have other suggestions.

Thanks for your help.

-mario

**Comment by nicolas2bert** [comment]
Hey Mario,
It looks like the issue is on the Docker side.
Please check this Getting Started with Docker for Windows : page https://docs.docker.com/docker-for-windows/

> Note: Docker for Windows requires 64bit Windows 10 Pro, Enterprise and Education (1511 November update, Build 10586 or later) and Microsoft Hyper-V. Please see What to know before you install for a full list of prerequisites.

For more support, please check the Docker forum for windows : https://forums.docker.com/c/docker-for-windows

Nicolas

**Comment by Mariosworld11** [comment]
Thank you for your response.

I did see this document previously. The instructions for Scality mention to install the Docker ToolBox. There is a note in the document that you mention where you can use the Oracle VirtualBox instead of Hyper-V. I will try installing Hyper-V to see if it solves the issue.

Thanks again for pointing that out.

-mario

**Comment by Mariosworld11** [comment]
For the past few days I was able to narrow down where the issue might be coming from. It seems that VM that gets created in the Oracle VirtualBox Manager is a Linux 64 bit VM. In my environment, if I open this manager, I am not able to create a 64 bit VM. It only shows 32 bit virtual machines. I did follow some of the suggestions at the Oracle support site regarding this issue (on a physical machine) but have not been successful. I do not believe that they support installation of their software on a VM. I was curious if anyone else has been able to get this to work on a Windows Server 2008 or 2012/r2 Virtual Machine. 

I have been able to get this to work on a Windows Server 2012r2  physical machine.

Thanks for your help.

-mario
